recipes/libnetcdf/bld.bat

```diff
@@ -0,0 +1,26 @@
+mkdir %SRC_DIR%\build
+cd %SRC_DIR%\build
+
+
+set LIB=%LIBRARY_LIB%;%LIB%
+set LIBPATH=%LIBRARY_LIB%;%LIBPATH%
+set INCLUDE=%LIBRARY_INC%;%INCLUDE%
+set HDF5_DIR=%LIBRARY_PREFIX%\cmake
+
+cmake -G "NMake Makefiles" -DCMAKE_BUILD_TYPE=Release -DCMAKE_PREFIX_PATH=%LIBRARY_PREFIX% -DCMAKE_INSTALL_PREFIX:PATH=%LIBRARY_PREFIX% %SRC_DIR%
```

Comment:
Not necessarily we do have `cmake` pre-installed on AppVeyor. Not sure what is the best way to go though.